### PR TITLE
Update documentation for changing board shortcut #946

### LINKED ISSUE #946: Update documentation for changing board shortcut
Follow-up for #919

### PR DESCRIPTION
**Pull request by HyungJon**
Fixes #946

### CODE REVIEW

File: docs/boards.md [hunk 1]
```diff
@@ -20,4 +20,8 @@
 
 ## Deleting a Board
 
-- Select the board you want to remove under `Boards > Delete`.
\ No newline at end of file
+- Select the board you want to remove under `Boards > Delete`.
+
+## Switching the board
+
+- Push Ctrl+B to quickly switch the currently open board to another saved board.
```
Comment (dariusf) [inline docs/boards.md]: You can use `<kbd>` tags here. See the panels document.

**Comment by dariusf** [comment]
Also you should update the keyboard shortcuts document.

File: docs/keyboardShortcuts.md [hunk 2]
```diff
@@ -31,16 +31,16 @@ These key sequences (one key followed by the other) are used to load (in the bro
 | ------------- |:-------------:| ------------|
 | Next panel (Forward)|<kbd>F</kbd>|Focus moves to the next panel on the right of the current panel. If there are no more panels on the right, goes back to the leftmost panel|
 | Next panel (Backward)|<kbd>D</kbd>|Similar to above, but moves focus in the reverse direction|
-| First issue of issue list|<kbd>Ctrl</kbd> + <kbd>↓</kbd> / <kbd>1</kbd>| Jumps to the first issue in issue list|
-| Issue list *to* filter box|<kbd>Ctrl</kbd> + <kbd>↑</kbd> / <kbd>Space</kbd> <kbd>Space</kbd>|Focus moves from the issue list to filter box of the same panel|
-| Filter box *to* issue list|<kbd>Ctrl</kbd> + <kbd>↓</kbd> / <kbd>Space</kbd> <kbd>Space</kbd>|Focus moves from the filter box to issue box of the same panel|
+| First issue of issue list|<kbd>Ctrl</kbd> + <kbd>�넃</kbd> / <kbd>1</kbd>| Jumps to the first issue in issue list|
```
Comment (dariusf) [inline docs/keyboardShortcuts.md]: Some stray unicode characters here. Your editor might be automatically converting them.

**Comment by HyungJon** [comment]
Is this good to merge now? I have to update it again before that though.

**Comment by dariusf** [comment]
Yes, it's fine. What do you need to update?

**Comment by HyungJon** [comment]
RuntimeExceptions causing Travis failure after unusually long time? Is this happening elsewhere as well?

**Comment by HyungJon** [comment]
Coverage check is taking unusually long. Can I merge this PR (before it becomes outdated again) since this is just about documentation?

**Comment by dariusf** [comment]
Yup. Could you squash? Then I'll just merge it. No need to wait for CI.

**Comment by HyungJon** [comment]
Right. Shall I do the merging this time, if only to get used to it now?

**Comment by dariusf** [comment]
If the build fails you may not be able to, but otherwise sure.

**Comment by HyungJon** [comment]
Is it okay to merge an out-of-date PR if it's unlikely to cause conflicts? Or should I merge master to this branch again?

**Comment by dariusf** [comment]
It's okay. In ambiguous cases we shouldn't, but this clearly won't cause conflicts.

Just rebase onto master to update in future. It's conceptually the same as merging, then squashing like you do.

**Comment by dariusf** [comment]
You should be able to merge this now.

**Comment by HyungJon** [comment]
Oh damn. I actually merged from master and from this PR in the wrong order, and caused last commit message to be the merge from master. It won't happen again.

**Comment by dariusf** [comment]
Please check before pushing. The state of the repo after pushing will be exactly the same as it is locally before you push. You can iron out such issues then. If you don't get the merge right the first time, `reset --hard HEAD^` and try it again.

**Pull request merged**